src/sonarwhal-theme/layout/partials/header.hbs

```diff
@@ -1,4 +1,26 @@
 <header class="header">
+    <div class="header-social">
+        <div class="header-social__container container">
+            <img src="/images/jsf_logo_white.png" alt="A JS Foundation Project" />
```

Comment:
`jsf_logo_white.png` => We should use the [SVG version](https://github.com/sonarwhal/sonarwhal.com/issues/327#issuecomment-363154300).